code/modules/maps/spawners/spawners.dm

```diff
@@ -1198,7 +1199,10 @@
 	/obj/item/weapon/reagent_containers/glass/bottle/chloralhydrate,
 	/obj/item/weapon/dnainjector/nofail/randompower,
 	/obj/item/weapon/gun/syringe/rapidsyringe,
-	/obj/item/voucher/free_item/medical_safe
+	/obj/item/voucher/free_item/medical_safe,
+	/obj/item/weapon/storage/pill_bottle/nanofloxacin,
+	/obj/item/weapon/gun/portalgun,
```

Comment:
Sure that you meant to put these in the medical spawner?

Reply:
woops